src/resources/app/store/store.js

```diff
@@ -0,0 +1,21 @@
+import Vue from "vue";
+import Vuex from "vuex";
+// import api from "./../api";
```

Comment:
perhaps it can be removed if it's unused(?)

Reply:
yes it will be cleaner if we remove any unused comment